Stockwatch/Stockwatch.Model/StockwatchDbContext.cs

```diff
@@ -4,8 +4,8 @@ namespace Stockwatch.Model
 {
     public class StockwatchDbContext : DbContext
     {
-        public DbSet<Stockdata> Stockdatas { get; set; }
-        public DbSet<Stocksymbol> Stocksymbols { get; set; }
+        public DbSet<StockData> StockDatas { get; set; }
```

Comment:
This name also would change to StockAlertRanges when model name changes, you would also have to add a migration

Reply:
I have changed it